packages/components/FilePreview/AudioView.tsx

```diff
@@ -192,8 +188,8 @@ export const AudioView: React.FC<{
               }}
               resizeMode="cover"
               style={{
-                height: THUMBNAIL_WIDTH,
-                width: THUMBNAIL_WIDTH,
+                height: 140,
+                width: 140,
```

Comment:
can you add back the constant pls? named variables are better than hardcoded numbers, you can add it in this file directly